x-pack/plugins/enterprise_search/public/applications/shared/role_mapping/constants.ts

```diff
@@ -107,3 +107,25 @@ export const MANAGE_ROLE_MAPPING_BUTTON = i18n.translate(
     defaultMessage: 'Manage',
   }
 );
+
+export const ROLE_MAPPINGS_TITLE = i18n.translate(
+  'xpack.enterpriseSearch.roleMapping.roleMappingsTitle',
+  {
+    defaultMessage: 'Users & roles',
+  }
+);
+
+export const EMPTY_ROLE_MAPPINGS_TITLE = i18n.translate(
+  'xpack.enterpriseSearch.roleMapping.emptyRoleMappingsTitle',
+  {
+    defaultMessage: 'No role mappings yet',
+  }
+);
+
+export const ROLE_MAPPINGS_DESCRIPTION = i18n.translate(
+  'xpack.enterpriseSearch.roleMapping.roleMappingsDescription',
```

Comment:
💯 Thanks for DRYing these out!

Just as an FYI, if our i18n localizers had already translated the old i18n ID key (`xpack.enterpriseSearch.workplaceSearch.roleMapping.roleMappingsDescription`), we would have needed to update the translation JSON files (https://github.com/elastic/kibana/tree/master/x-pack/plugins/translations/translations) to update the IDs (and also the i18n CI job would likely have thrown an error). It looks like they haven't translated this yet though, so we should be good!

Reply:
Thanks for the heads up. Thankfully CI fails and lets me know because I always forget 😂